### DISCUSSION TITLE
Can stack of interferograms from GMTSAR  be used in Mintpy

### DISCUSSION THREAD

**Discussion by zxzxzxzx12** [insarlab/MintPy#710] [Q&A]
Sorry, can stack of interferograms from GMTSAR  be used in Mintpy? there are example template files for various processors except for GMTSAR.
    I use 5.4.3 GMTSAR but there are not .LED .PRM and baseline.txt files said in Example data directory. When I runned the smallbaselineApp.py, it seem to meet problem in load data step. It raised problem in load_data.py, the problem is "date6s = readfile.read_attribute(dsPath0)['DATE12'].replace('_','-').split('-')"  "KeyError: 'DATE12' ". 
   I really want to use  Mintpy, but I want to know if really GMTSAR can be used in Mintpy as said in README.md , and if it can be used , are there any more detailed information that I missed.

**Comment by yunjunz** [comment]
Currently the only documentation related with `gmtsar + mintpy` workflow is here: https://mintpy.readthedocs.io/en/latest/dir_structure/#gmtsar. Manual preparation is needed in the gmtsar products, to get the directory structure as listed in the example, before running mintpy.

**Reply by zxzxzxzx12** [reply]
Thank you so much for your reply, yesterday I succeeded in obtaining velocity product.   There are not .PRM and .LED file in Merge dictory in GMTSAR. SO I have to use .PRM and .LED file in Track 1 F1/raw dictory instead. Besides, I run SAT_baseline command to obtain  baseline.txt manually  useing .PRM file. I meet  ValueError: could not broadcast input array from shape (835,1107) into shape (2070,3150) in  plot & save to pic, but this step is not necessary. This is my experience, but I don't know whether it is accurate since the information of tutorial is limited. I would be grateful if If anyone know the accurate steps and share it generously.
